backend/tests/integration/records/views/test_views_jobs.py

```diff
@@ -186,8 +186,6 @@ def test_jobs_record_search_results_returns_pending_job_to_superuser(
     expected_metadata.pop("_created")
     expected_results = 1
 
-    assert result.json["hits"]["total"] == expected_results
```

Comment:
why remove this?

Reply:
the same check is on line 194 (192 after change) - 4 lines under this one